.idea/workspace.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you remove this file and add `/.idea` to the `.gitignore` file? This is not needed for the website.

Reply:
Sure.. Will do it